packs/spell-effects/spell-effect-dragon-form-mirage.json

```diff
@@ -0,0 +1,230 @@
+{
+    "_id": "nWEx5kpkE8YlBZvy",
```

Comment:
This id looks new, and we have 3 effects being deleted that don't have replacement effects?

Reply:
That ID is from `spell-effect-dragon-form-green`, which is one of the "deleted" files.
There were 10 dragon forms on the spell from the CRB (black, blue, brass, bronze, copper, gold, green, red, silver, white); The PC1 version only has 8 (adamantine, conspirator, diabolic, empyreal, fortune, horned, mirage, omen). I deleted silver and white, and reused the IDs from the others in alphabetical order.

Reply:
I'm not sure why Github likes to say that different files were modified than were sometimes. It should be:
Black > Adamantine
Blue > Conspirator
Brass > Diabolic
Bronze > Empyreal
Copper > Fortune
Gold > Horned
Green > Mirage
Red > Omen